### PR TITLE
GlobalUserPage + GlobalPreferences per T1681

### PR DESCRIPTION
**Pull request by MacFan4000**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -3313,7 +3314,21 @@
...
+	),
 
+ 	//GlobalUserPage
+ 	'wgGlobalUserPageAPIUrl' => array(
+		'default' => "https://meta.miraheze.org",
+	),
+	'wgGlobalUserPageDBname' => array(
+		'default' => 'metawiki',
+	),
+	$wgDefaultUserOptions['globaluserpage'] = false;
```
Comment (MacFan4000) [inline LocalSettings.php]: @JohnFLewis how should this be done? Reception123 says they are not sure.

Comment (MacFan4000) [inline LocalSettings.php]: @JohnFLewis how should this be done? @Reception123 says they are not sure.

**Comment by JohnFLewis** [comment]
See the other examples of this.

**Comment by MacFan4000** [comment]
@JohnFLewis I don't see anything else related to $wgDefaultUserOptions

**Comment by JohnFLewis** [comment]
https://github.com/miraheze/mw-config/blob/master/LocalSettings.php#L7293-L7295

File: LocalSettings.php [hunk 2]
```diff
@@ -506,6 +506,7 @@
 			"$IP/extensions/FlaggedRevs/backend/schema/mysql/FlaggedRevs.sql",
 			"$IP/extensions/Flow/flow.sql",
 			"$IP/extensions/GlobalBlocking/localdb_patches/setup-global_block_whitelist.sql",
+			"$IP/extensions/GlobalPreferences/sql/tables.sql",
```
Comment (JohnFLewis) [inline LocalSettings.php]: This isn’t something we want on all wikis.

The table shouldn’t even be in any wiki.

**Pull request merged**